### DISCUSSION TITLE
Renovate must allow changing of the PR's Title

### DISCUSSION THREAD

**Discussion by maerean** [renovatebot/renovate#23450] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest?

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

when we change the Title of PRs created by renovate-app, the bot changes the title back.

I understand that Renovate uses the title as a cache key.

This is an incorrect behaviour of Renovate.

Renovate must use immutable properties (like the PR's number) for the cache key.

By definition, the PR's title is mutable.

Just to provide all the information, we prefix all PR titles with a "[JIRA-ID]" text.
Regardless, Renovate can and must work regardless of the PR's title.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment]
no, not supported, renovate always updates the title and it's not a cache key

**Reply by Chumper** [reply]
In that case we should update the documentation here: https://docs.renovatebot.com/key-concepts/pull-requests/#branch-name-and-pr-title-are-cache-keys

**Reply by viceice** [reply]
ok, they are not directly cache keys, but they are relevant detect existing updated.

**Reply by Chumper** [reply]
Got it, I was just afraid to throw around false facts.

**Reply by rarkins** [reply]
Closing due to rude assertion "This is an incorrect behaviour of Renovate."
